src/server/controllers/user.js

```diff
@@ -8,28 +8,37 @@ const jwtSecret = 'mysecret';
 const register = async (req, res) => {
     const { username, password } = req.body;
 
-    const createdUser = null;
+    const hashedPassword = await bcrypt.hash(password, 10)
+    const createdUser = await prisma.user.create({
+        data: {
+            username: username,
+            password: hashedPassword
+        }
+    })
 
     res.json({ data: createdUser });
 };
 
 const login = async (req, res) => {
     const { username, password } = req.body;
 
-    const foundUser = null;
+    const foundUser = await prisma.user.findUnique({
+        where: {
+            username: username
+        }
+    })
 
     if (!foundUser) {
         return res.status(401).json({ error: 'Invalid username or password.' });
     }
 
-    const passwordsMatch = false;
+    const passwordsMatch = bcrypt.compareSync(password, foundUser.password);
 
     if (!passwordsMatch) {
         return res.status(401).json({ error: 'Invalid username or password.' });
     }
 
-    const token = null;
-
+    const token = jwt.sign(username, jwtSecret)
```

Comment:
maybe await needed here for the jwt.sign call?

Reply:
I don't think so